plugins/inputs/cisco_telemetry_mdt/cisco_telemetry_mdt.go

```diff
@@ -50,13 +52,171 @@ type CiscoTelemetryMDT struct {
 
 	// Internal state
 	aliases   map[string]string
+        dmes      map[string]string
 	warned    map[string]struct{}
 	extraTags map[string]map[string]struct{}
+	nxpathMap map[string]map[string]string //per path map
+	nxpropMap map[string]int //Global property map
 	mutex     sync.Mutex
 	acc       telegraf.Accumulator
 	wg        sync.WaitGroup
 }
 
+type JsonStructure struct {
```

Comment:
This is a really bad struct name especially due to the fact that it is exported! So please choose a name that describes what the structure contains and make it private.